src/weights.jl

```diff
@@ -193,6 +193,61 @@ pweights(vs::RealArray) = ProbabilityWeights(vec(vs))
     end
 end
 
+"""
+    eweights(t::AbstractVector{<:Integer}, λ::Real)
+    eweights(t::AbstractVector{T}, r::StepRange{T}, λ::Real) where T
+    eweights(n::Integer, λ::Real)
+
+Construct [`Weights`](@ref) vector which assigns exponentially decreasing weights to past
+observations, which in this case corresponds to larger integer values `i` in `t`.
+
```

Comment:
`n` should be mentioned here I think. I suggest something like:
```suggestion
If an integer `n` is provided, weights are generated for values from 1 to `n`
(equivalent to `t = 1:n`).

```